core/asset/block.go

```diff
@@ -32,11 +32,16 @@ func (reg *Registry) indexAnnotatedAsset(ctx context.Context, a *Asset) error {
 	m := map[string]interface{}{
 		"id":               a.AssetID,
 		"alias":            a.Alias,
-		"definition":       a.Definition,
+		"raw_definition":   json.HexBytes(a.RawDefinition()),
+		"vm_version":       a.VMVersion,
 		"issuance_program": json.HexBytes(a.IssuanceProgram),
 		"tags":             a.Tags,
 		"is_local":         "no",
 	}
+	adef, err := a.Definition()
+	if err == nil {
+		m["definition"] = adef
```

Comment:
Setting this conditionally seems like a user-visible change. Under what circumstances would it be omitted (when a.Definition returns an error)? Should we fill in a default value in that case? How would that case have manifested before this patch?

cc @jeffomatic